Daspren/daspren-parad/tests/file_opened.json

```diff
@@ -0,0 +1,47 @@
+{
+  "input": {
+    "message": "{\"time\":\"2024-03-04T10:43:31.337737900+01:00\",\"level\":\"INFO\",\"message\":\"File opened\",\"event_data\":{\"basename\":\"AppData\",\"created_length\":\"0\",\"executable\":\"C:\\Windows\\System32\\RuntimeBroker.exe\",\"executable_basename\":\"RuntimeBroker.exe\",\"executable_category\":\"System\",\"fullpath\":\"C:\\Users\\Testeur\\AppData\",\"fullpath_category\":\"User\",\"hash\":\"6b1070d18921913ef663562e238873717fff4f2afe3e7672c226e91adc614dea\",\"pid\":\"4960\",\"ppid\":\"0\",\"signed\":\"true\"},\"channel\":\"remote\",\"env_id\":\"7ba0a633-f8a3-410b-ba6f-5974705ced3a\",\"parad_version\":\"0.6.1\",\"os\":\"Windows 10 Pro\",\"machine_name\":\"bloquant\",\"msg_id\":6094}",
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Daspren Parad",
+        "dialect_uuid": "23813540-b658-48dd-b030-e9b92168bbf4"
+      }
+    }
+  },
+  "expected": {
+    "message": "{\"time\":\"2024-03-04T10:43:31.337737900+01:00\",\"level\":\"INFO\",\"message\":\"File opened\",\"event_data\":{\"basename\":\"AppData\",\"created_length\":\"0\",\"executable\":\"C:\\Windows\\System32\\RuntimeBroker.exe\",\"executable_basename\":\"RuntimeBroker.exe\",\"executable_category\":\"System\",\"fullpath\":\"C:\\Users\\Testeur\\AppData\",\"fullpath_category\":\"User\",\"hash\":\"6b1070d18921913ef663562e238873717fff4f2afe3e7672c226e91adc614dea\",\"pid\":\"4960\",\"ppid\":\"0\",\"signed\":\"true\"},\"channel\":\"remote\",\"env_id\":\"7ba0a633-f8a3-410b-ba6f-5974705ced3a\",\"parad_version\":\"0.6.1\",\"os\":\"Windows 10 Pro\",\"machine_name\":\"bloquant\",\"msg_id\":6094}",
```

Comment:
We detected that the json in `message` seems to be malformed. Can you check it? Thanks.
According to our analysis, it seems to be the backslashes in the fields `executable` and `fullpath` that generate the issue.